Vagrantfile

```diff
@@ -5,14 +5,38 @@
 # Chef server.
 # See http://www.vagrantup.com/ for info on Vagrant.
 
-$local_environment = "Test-Laptop"
+require 'json'
+
+if File.basename(File.expand_path(".")) == "vbox"
```

Comment:
What is this trying to do? A comment would be useful if it's for what I think it is.